### DISCUSSION TITLE
flink查询OceanBase表是否有做谓词下推？

### DISCUSSION THREAD

**Discussion by tiloulou177** [oceanbase/flink-connector-oceanbase#28] [Q&A] [answered]
hi
  I am wondering is there Push-down when using this connector in flink  to query OceanBase table ?

**Comment by whhe** [comment, marked as answer]
This connector only provides `SinkWriter` implementation for now, and so there is no predicate pushdown. We plan to provide lookup feature in the near future and will consider to take this work at that time.

**Reply by tiloulou177** [reply]
I see. Thanks for replying so fast.
